### PR TITLE
new release

### PR DESCRIPTION
**Pull request by caneleex**

### CODE REVIEW

File: config.json.example [hunk 1]
```diff
@@ -12,6 +12,8 @@
     "top_gg_token": "your https://top.gg token",
     "discord_extreme_list_token": "your https://discordextremelist.xyz token",
     "discord_boats_token": "your https://discord.boats token",
+    "signing_key": "",
+    "backend_port": "",
```
Comment (caneleex) [inline config.json.example:16]: i know it's kinda obvious but i think there should be comments on these two new settings

File: gradle.properties.example [hunk 2]
```diff
@@ -0,0 +1,5 @@
+KITTYBOT_DB_HOST=
+KITTYBOT_DB_PORT=
+KITTYBOT_DB_DATABASE=
+KITTYBOT_DB_USER=
+KITTYBOT_DB_PASSWORD=
```
Comment (caneleex) [inline gradle.properties.example:5]: same here

File: config.json.example (hunk 1 above)
Comment (TopiSenpai) [reply config.json.example:16]: uh yes there should

File: gradle.properties.example (hunk 2 above)
Comment (TopiSenpai) [reply gradle.properties.example:5]: yesss you're right

File: build.gradle [hunk 3]
```diff
@@ -34,13 +34,14 @@ dependencies {
     implementation("com.github.KittyBot-Org:Lavalink-Client:f7842dfdb4") {
         exclude group: 'com.sedmelluq', module: 'lavaplayer'
     }
-    implementation 'com.github.devoxin:lavaplayer:1.3.59'
+    implementation 'com.sedmelluq:lavaplayer:1.3.53'
```
Comment (TopiSenpai) [inline build.gradle:37]: we can go to devoxin's fork again cuz it has some NPE fixes...

Comment (caneleex) [reply build.gradle:37]: ffs, this is like threading on thin ice

Comment (TopiSenpai) [reply build.gradle]: yeah it's so stupid seq should add him as collab & then we have one up2date project again
I'm wirting a message in lavaplayer channel now 
already talked with devoxin about it & we is open for it...

**Comment by caneleex** [comment]
after we merge dependencies branch/pull request into development, try regenerating jooq files to see if our gradle-jooq-plugin fork + jooq 3.14.1 work together

**Review by TopiSenpai — APPROVED** [review]
lgtm

**Pull request merged**